### PR TITLE
waybar: 0.9.5 -> 0.9.7

### PR DESCRIPTION
**Pull request by Synthetica9**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119561` at df4e7e68 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>waybar</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/meson-cmake.md">meson-cmake</a>

Meson uses CMake as a fallback dependency resolution method and it likely is not necessary here. The message about cmake not being found is purely informational.

Near pkgs/applications/misc/waybar/default.nix:40:3:
```
   |
40 |   nativeBuildInputs = [
   |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `fetchpatch`.
Near pkgs/applications/misc/waybar/default.nix:4:3:
```
  |
4 | , fetchpatch
  |   ^
```
</li>
  </ul>
</details>

File: pkgs/applications/misc/waybar/default.nix [hunk 1]
```diff
@@ -28,26 +28,15 @@
...
-    sha256 = "1kzrgqaclfk6gcwhknxn28xl74gm5swipgn8kk8avacb4nsw1l9q";
+    sha256 = "17cn4d3dx92v40jd9vl41smp8hh3gf5chd1j2f7l1lrpfpnllg5x";
   };
 
-  patches = [
-    # XXX: REMOVE ON NEXT VERSION BUMP
-    # Fixes compatibility of the bluetooth and network modules with linux kernel
-    # >=5.11
-    # c.f. https://github.com/Alexays/Waybar/issues/994
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/waybar/default.nix:45]: Please remove fetchpatch from inputs.

**Review by minijackson — APPROVED** [review]
Compiles and runs fine, thanks!

---

Result of `nix-review pr 119561` [1](https://github.com/Mic92/nix-review)
<details>
  <summary>1 package were built:</summary>

  - waybar
</details>

**Review by Technical27 — APPROVED** [review]
Testing this change for a bit, seems to work fine.

**Pull request merged**